### ISSUE TITLE
Unavailable shared library

### ISSUE THREAD

**Issue by cco3** [pimterry/lambda-git#21]
I'm getting the following error:
`git: error while loading shared libraries: libpcre.so.0: cannot open shared object file: No such file or directory`

**Comment by pimterry** [comment]
Yes, in short, I think this library should probably be deprecated now, for most cases, and you should use layers instead, e.g. https://github.com/lambci/git-lambda-layer. Lots of shared libraries are not longer included in AWS's modern base images, so there's quite a lot of separate setup required otherwise.

There's more detail about this over here that might be useful: https://github.com/pimterry/lambda-git/issues/14

I'm going to leave this open until I get around to going through and formally documenting that.

**Comment by cco3** [comment]
Any suggestions for if layers are unavailable (e.g., I'm using lambda through netlify)?

**Comment by pimterry** [comment]
No idea, sorry, that's probably a question best asked to the netlify team.
